### PR TITLE
Wfs-error-check

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

File: geoengine/workflow.py [hunk 1]
```diff
@@ -165,17 +165,22 @@ def get_dataframe(self, bbox: QueryRectangle) -> gpd.GeoDataFrame:
...
             and `end`.
             '''
 
-            data = gpd.read_file(StringIO(data_response.text))
+            # data = gpd.read_file(StringIO(data_response.text))
+            data = gpd.GeoDataFrame.from_features(geo_json)
             data = data.set_crs(bbox.srs, allow_override=True)
 
-            geo_json = data_response.json()
+            # geo_json = data_response.json()
```
Comment (jdroenner) [inline geoengine/workflow.py:183]: remove comment

File: geoengine/workflow.py [hunk 2]
```diff
@@ -165,17 +165,22 @@ def get_dataframe(self, bbox: QueryRectangle) -> gpd.GeoDataFrame:
...
+
+        def geo_json_with_time_to_geopandas(geo_json):
             '''
             GeoJson has no standard for time, so we parse the when field
             separately and attach it to the data frame as columns `start`
             and `end`.
             '''
 
-            data = gpd.read_file(StringIO(data_response.text))
+            # data = gpd.read_file(StringIO(data_response.text))
```
Comment (jdroenner) [inline geoengine/workflow.py:179]: remove comment

File: setup.cfg [hunk 3]
```diff
@@ -18,7 +18,7 @@ package_dir =
 packages = find:
 python_requires = >=3.7
 install_requires =
-    cartopy
+    cartopy==0.19.0.post1
```
Comment (jdroenner) [inline setup.cfg:21]: is this related to this PR?

File: geoengine/workflow.py (hunk 2 above)
Comment (ChristianBeilschmidt) [reply geoengine/workflow.py]: 8cabd02
